### ISSUE TITLE
Output format "markdown"

### ISSUE THREAD

**Issue by estahn** [HunterMcGushion/docstr_coverage#133]
I'm using the output to be appended as a PR comment, hence proper formatting as markdown would be ideal. Possibly more formats would be useful for others to do post-processing.

Formats I think could be useful: `text` (current), `json`, `markdown`, `github-markdown`

**Comment by MiWeiss** [comment]
Thanks @estahn for opening the PR.

This is definitely something we'd like to support, and also something we have forseen for a while. However, doing this properly may cost quite some time and I don't have that time to spare ;-)

However, there's already quite some preparation done for this functionality. The printer.py file contains a class which is used for printing the results to the command line. See https://github.com/HunterMcGushion/docstr_coverage/blob/master/docstr_coverage/printers.py. Our long term plan is to extract an abstract superclass from this class and to provide multiple exporters (e.g. JSON file, Markdown printout, ...). 

Would you be interested in contributing this feature, i.e., in writing a markdown printer? @HunterMcGushion  and I would of course be happy to help/assist and review whenever needed.

**Comment by estahn** [comment]
@MiWeiss Yes, I already started on something. Just curious why you use `logger.info` to print out the text.

**Comment by MiWeiss** [comment]
Puh, if I recall correctly this was an open todo long before I joined the project which was then implemented along some refactoring.

I can only assume that the reason was to facilitate output configuration. But afaik, this is not really used by any of our users; it's not something we documented, nor something for which I'd see a particuarly obvious use case.

Thus, if it facilitates your contribution, I'd be happy to switch back to `print()`. Of course, @HunterMcGushion will have his veto right, thus it would probably be best to submit that change in a separate PR.

**Comment by skv0zsneg** [comment]
Hi! Is this issue still in developing?

In my working cases I use `get_docstring_coverage` to put coverage information on reStructuredText pages throw Sphinx. But it forced me to use some non beautiful hacks. I think that just by adding some flag like `print_output` in arguments of this function and to argparse for creating coverage report in reStructuredText or Mardown format will be great feature!

If you liked this idea I will do it with pleasure😃
